snap/hooks/configure

```diff
@@ -431,6 +431,19 @@ then
   need_api_restart=true
 fi
 
+# scheduler --address flag is removed after 1.24
+if grep -e "--address=" ${SNAP_DATA}/args/kube-scheduler
+then
+  "${SNAP}/bin/sed" -i '/--address=/d' "$SNAP_DATA/args/kube-scheduler"
```

Comment:
Minor change. I think we need a `need_api_restart=true` after this line too.

Reply:
this flag is already deprecated and has no effect, so I think we can avoid restarting the API server.